CUTSCENE/CanSetExitStateForCamera.md

```diff
@@ -12,6 +12,6 @@ Whether or not it is safe to run functions on the camera,
 as the camera is now no longer being used by the cutscene.
 
 ## Parameters
-* **p0**: Mostly false
+* **p0**: 
```

Comment:
This can be employed while cutting the camera in the exit state to prevent problems with unregistered entities persisting on screen for a frame.